packages/smarthr-ui/src/components/Layout/Container/stories/Container.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import { Base } from '../../../Base'
+import { Container, classNameGenerator } from '../Container'
+
+import type { Meta, StoryObj } from '@storybook/react'
+import { ComponentProps, useLayoutEffect, useRef, useState } from 'react'
+import { Stack } from '../../Stack'
+import { DeviceProvider } from '../../../..'
+
+const meta = {
+  title: 'Components/Layout/Container',
+  component: Container,
+  render: ({ size, children, ...rest }) => (
+    <Container {...rest} size={size}>
+      {children ?? (
+        <DisplayDimensionsBase padding={1.5} size={size}>
+          Story をわかりやすくするためのカスタム Base
+        </DisplayDimensionsBase>
+      )}
+    </Container>
+  ),
+  decorators: [
+    (Story) => (
+      <DeviceProvider>
+        <Story />
+      </DeviceProvider>
+    ),
+  ],
+  parameters: {
+    chromatic: { disableSnapshot: true },
+  },
+} satisfies Meta<typeof Container>
+export default meta
```

Comment:
```suggestion
export default {
  title: 'Components/Layout/Container',
  component: Container,
  render: ({ size, children, ...rest }) => (
    <Container {...rest} size={size}>
      {children ?? (
        <DisplayDimensionsBase padding={1.5} size={size}>
          Story をわかりやすくするためのカスタム Base
        </DisplayDimensionsBase>
      )}
    </Container>
  ),
  decorators: [
    (Story) => (
      <DeviceProvider>
        <Story />
      </DeviceProvider>
    ),
  ],
  parameters: {
    chromatic: { disableSnapshot: true },
  },
} satisfies Meta<typeof Container>
```
でかけるかなと！ (そのためのsatisfiesなので

Reply:
後続で `meta.render` を使ってるんですが、render の中身を切り出すべき?

Reply:
修正しました! [a35c780](https://github.com/kufu/smarthr-ui/pull/5704/commits/a35c78067fd04a9f6922818ad14f857a2b4ccff6)